README.md

```diff
@@ -96,4 +88,4 @@ Hint: when in doubt, use **Administrator**
 | Caller        | testcaller@example.com | password | 999999   |
 | Basic user    | testuser@example.com   | password | 999999   |
 
-The SMS one-time password is only required in some parts of the app.
+The SMS one-time password is only required in some parts of the app. does this make sense? 
```

Comment:
I don't understand the addition of "does this make sense?" but I'm guessing it's a mistake.